internal/provider/waypoint/data_source_waypoint_add_on.go

```diff
@@ -42,6 +42,28 @@ func (d *DataSourceAddOn) Metadata(ctx context.Context, req datasource.MetadataR
 	resp.TypeName = req.ProviderTypeName + "_waypoint_add_on"
 }
 
+// AddOnDataSourceModel describes the data source data model.
+type AddOnDataSourceModel struct {
+	ID             types.String `tfsdk:"id"`
+	Name           types.String `tfsdk:"name"`
+	ProjectID      types.String `tfsdk:"project_id"`
+	OrgID          types.String `tfsdk:"organization_id"`
+	Summary        types.String `tfsdk:"summary"`
+	Labels         types.List   `tfsdk:"labels"`
+	Description    types.String `tfsdk:"description"`
+	ReadmeMarkdown types.String `tfsdk:"readme_markdown"`
+	CreatedBy      types.String `tfsdk:"created_by"`
+	Count          types.Int64  `tfsdk:"install_count"`
+	Status         types.Int64  `tfsdk:"status"`
+	ApplicationID  types.String `tfsdk:"application_id"`
+	DefinitionID   types.String `tfsdk:"definition_id"`
+	OutputValues   types.List   `tfsdk:"output_values"`
+
+	TerraformNoCodeModule types.Object `tfsdk:"terraform_no_code_module"`
+
+	InputVars types.Set `tfsdk:"input_variables"`
+}
+
 //TODO: Make sure this schema is correct (do we want to include count or output values?)
```

Comment:
```suggestion
```

nit: I think we can remove this line now, yeah?

Reply:
I think so too!